app/src/main/java/com/lastaoutdoor/lasta/ui/components/ShareSheets.kt

```diff
@@ -34,7 +34,7 @@ fun shareActivity(activity: Activity, context: Context) {
   val textFromActivity =
       context.getString(R.string.share_body) +
           "\n${activity.name} \n" +
-          context.getString(R.string.share_base_url) +

```

Comment:
Please add the URL of the current release of the app

Reply:
I did it ! will have to be changed though